src/components/WebexMeeting/WebexMeeting.jsx

```diff
@@ -42,21 +48,29 @@ export default function WebexMeeting({className, meetingID, style}) {
         {isActive
           ? <WebexInMeeting meetingID={ID} />
           : <WebexInterstitialMeeting meetingID={ID} />}
+        {showSettings && (
+          <Modal
+            onClose={() => adapter.meetingsAdapter.toggleSettings(ID)}
+            title="Settings"
+          >
+            <WebexSettings meetingID={ID} />
+          </Modal>
+        )}
+        {showRoster && isActive && (
```

Comment:
isActive is surely true, look on line 48

Reply:
done